pkg/restic/restic_test.go

```diff
@@ -464,6 +463,38 @@ func TestBackupRestoreWithArgs(t *testing.T) {
 	}
 }
 
+func TestBackupWithTimeout(t *testing.T) {
+	tempDir, err := os.MkdirTemp("", "stash-unit-test-")
```

Comment:
```go
tempDir, err := os.MkdirTemp("", "kubestash-unit-test-")
```

Reply:
All other tests contain `stash-unit-test-`. Kept this for the `homogeneous` dir name for all tests.